src/applications/personalization/profile/tests/components/contact-information/PersonalInformation.update-address.unit.spec.jsx

```diff
@@ -254,7 +254,7 @@ describe('Updating', () => {
       it('should show an error and not auto-exit edit mode if the transaction cannot be created', async () => {
         await testTransactionCreationFails(addressName);
       });
-      it('should show an error and not auto-exit edit mode if the transaction fails quickly', async () => {
+      it.skip('should show an error and not auto-exit edit mode if the transaction fails quickly', async () => {
```

Comment:
TIL about `.skip()`! are you planning on unskipping this soon?

Reply:
nvm! just saw your comment about flaky tests

Reply:
I am hoping to fix it soon.

I made this ticket so it doesn't get forgotten. I'll try and work it into unplanned work this sprint if I have time. It passes locally, just flaky in the CI/CD pipelines
https://github.com/department-of-veterans-affairs/va.gov-team/issues/77636